src/components/sbb-divider/sbb-divider.scss

```
@@ -0,0 +1,25 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --_divider-color: var(--sbb-color-iron-default);
+  --_divider-background-color: var(--footer-divider-color, var(--sbb-color-cloud-default));
+}
+
+.sbb-divider {
+  display: block;
+
+  &--horizontal {
+    border-top: 0.5px solid var(--_divider-color);
```

Never use px directly. Also shouldn't it be 1 pixel?
```suggestion
    border-top: #{px-to-rem-build(1)} solid var(--_divider-color);
```